style.css

```diff
@@ -559,3 +566,241 @@ li a {
   align-items: center;
   width: 150px;
 }
+
+.content-container {
+  display: grid;
+  grid-template-columns: 100%;
+}
+
+.hide {
+  display: none;
+}
+
+.big-form-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.icons3 {
+  margin-top: 100px;
+}
+
+@media screen and (min-width: 991px) {
```

Comment:
Media query to low

- [ ] Kindly select a value from `992 - 1100px` to target desktop screens